pkg/reconciler/buildrun/buildrun.go

```diff
@@ -432,3 +443,21 @@ func (r *ReconcileBuildRun) createTaskRun(ctx context.Context, serviceAccount *c
 
 	return generatedTaskRun, nil
 }
+
+type patchStringValue struct {
+	Op    string `json:"op"`
+	Path  string `json:"path"`
+	Value string `json:"value"`
+}
+
+func (r *ReconcileBuildRun) patchTaskRun(ctx context.Context, tr *v1beta1.TaskRun, op, path, value string, opts metav1.PatchOptions) error {
+	payload := []patchStringValue{{
+		Op:    op,
+		Path:  path,
+		Value: value,
+	}}
+	data, _ := json.Marshal(payload)
```

Comment:
```suggestion
	data, err := json.Marshal(payload)
	if err != nil {
		return err
	}
```

Just in case we eventually have some static code analyzer that complains about ignored errors.